lib/widget/helpers/widgets.dart

```diff
@@ -214,22 +214,22 @@ class BoxWrapper extends StatelessWidget {
                           top: getBorderSide(
                             context,
                             boxController.borderWidth?.top,
-                            boxController.borderColor?.top,
+                            boxController.borderColor.top,
```

Comment:
does your Editor not complain about null safety? User can set borderWidth for example but exclude borderColor, and that will still show the border with default color. Please fix this.